### PR TITLE
SM: Display first and last name instead of UUID in from

### PR DESCRIPTION
**Pull request by warrenbailey**
In Secure message display first and last name in the inbox view rather than UUID

### CODE REVIEW

File: response_operations_ui/views/messages.py [hunk 1]
```diff
@@ -126,13 +126,21 @@ def _refine(message):
         'ru_ref': message.get('ru_id'),
         'business_name': message.get('@ru_id').get('name'),
         'subject': message.get('subject'),
-        'from': message.get('msg_from'),
-        'to': _get_name_from_message(message),
+        'from':_get_from_name(message),
```
Comment (JamesGardiner) [inline response_operations_ui/views/messages.py]: missing some whitespace here

File: response_operations_ui/views/messages.py [hunk 2]
```diff
@@ -126,13 +126,21 @@ def _refine(message):
...
     }
 
 
-def _get_name_from_message(message):
+def _get_from_name(message):
+    try:
+        msg_from = message['@msg_from']
+        return f"{msg_from.get('firstName')} {msg_from.get('lastName')}"
+    except KeyError:
+        return message.get('msg_from')
```
Comment (JamesGardiner) [inline response_operations_ui/views/messages.py]: return None? More efficient than trying to access the dict object again.

Comment (warrenbailey) [reply response_operations_ui/views/messages.py]: It's return the uuid (same as the method below)?

Comment (JamesGardiner) [reply response_operations_ui/views/messages.py]: Ah I missed the '@'

**Pull request merged**